lighthouse-core/gather/driver.js

```diff
@@ -701,7 +701,7 @@ class Driver {
 
     this.evaluateScriptOnLoad(`
         ${globalVarToPopulate} = new Set();
-        (${funcName} = ${funcBody}(${funcName}, ${globalVarToPopulate}))`);
+        (${funcName} = ${funcBody}(${funcName}, ${globalVarToPopulate}, window.__nativeError))`);
```

Comment:
should we inject this here or just have the script assume `window.__nativeError` exists like [`evaluateAsync` does](https://github.com/GoogleChrome/lighthouse/blob/0dff5efef196c5d9ecfab223cd1a54b1e2b169de/lighthouse-core/gather/driver.js#L152)? I'm not sure which is better, but it may be a bit cleaner to just use `window.__nativeError` inside `captureJSCallUsage`